templates/web/common/site_layout.tt.html

```diff
@@ -64,6 +64,7 @@
 
 									<li><label>[% lang("resources") %]</label></li>
 									<li>
+										<li><a href="https://www.youtube.com/playlist?list=PLjAH-USadsF3xgSVfUUBS4we3XBr-1r55" target="_blank">lang("video_tutorials")</a></li>
```

Comment:
This part of the code was recently moved by @MonalikaPatnaik in another file: web/common/producers_resources_list.tt.html , so we should add this new list item there.

Reply:
Done!